.werft/observability/install-satellite.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -euo pipefail
+
+SCRIPT_PATH=$(realpath "$(dirname "$0")")
+
+if [[ -z "${PREVIEW_NAME}" ]]; then
+  echo "Must set PREVIEW_NAME variable" 1>&2
+  exit 1
+fi
+
+if [[ -z "${KUBE_PATH}" ]]; then
+  echo "Must set KUBE_PATH variable" 1>&2
+  exit 1
+fi
+
+# exports all vars
+shopt -os allexport
+
+kubectl --kubeconfig "${KUBE_PATH}" create ns monitoring-satellite || true
+
+if ! command -v envsubst; then
+  go install github.com/a8m/envsubst/cmd/envsubst@latest
+fi
+
+obsDir="${SCRIPT_PATH}/observability"
+mkdir -p "${obsDir}"
+git clone https://roboquat:"$(cat /mnt/secrets/monitoring-satellite-preview-token/token)"@github.com/gitpod-io/observability.git "${obsDir}"
```

Comment:
We should replace this with `go install`, if this fixes the issue https://github.com/gitpod-io/gitpod/pull/12703